runtime/battery-station/src/lib.rs

```diff
@@ -78,7 +78,6 @@ use sp_runtime::{
 use nimbus_primitives::CanAuthor;
 use sp_version::RuntimeVersion;
 
-#[cfg(feature = "parachain")]
```

Comment:
https://github.com/zeitgeistpm/zeitgeist/blob/54099511b3819507fd0306db98fc7b96b1240b79/runtime/battery-station/src/integration_tests/xcm/mod.rs#L17

Ok, so the parachain feature is only required for the xcm crate. But we could have other modules inside `integration_tests` which should be able to run in standalone mode.

Reply:
Yes, in addition the logic applied here was plainly wrong. It would use the module when either `parachain`, `test` or both are set. The correct way to express that both feature have to be set would be like in the code you referenced.